_projects/polar-computing.md

```diff
@@ -4,9 +4,21 @@ collection: projects
 permalink: /projects/polar-computing
 abstract: "Analyzing opportunities and barriers in the uptake of High-Performance and Distributed Computing methods and cyberinfrastructures for Polar Science."
 status: active
+logo: prcn_logo.jpg
 ---
 
-The Polar Research Coordination Network connects the Polar Science, Data and High-Performance and Distributed Computing (HPDC) communities to enable deeper penetration of computing methods and cyberinfrastructure into the polar sciences. The potential for global impact with huge socio-economic costs makes understanding polar climate change, and the links between polar and global systems, and urgent priority. Despite the data- and compute-intensive scientific needs of modern polar science, use of HPDC is currently limited. Bringing these communities together in a sustained, multi-dimensional engagement has the potential to inform HPDC development and transform polar science research.
-
-<a href="http://polar.crc.nd.edu/"><i></i>Website</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1541620"><i class="fa fa-nsf"></i>NSF Award</a><br>
\ No newline at end of file
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/prcn.jpg" width="285">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```